parachains/runtimes/assets/statemine/src/weights/xcm/pallet_xcm_benchmarks_generic.rs

```diff
@@ -0,0 +1,137 @@
+	// Storage: ParachainSystem HostConfiguration (r:1 w:0)
+	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
+	pub(crate) fn query_holding() -> Weight {
+		(694_466_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(6 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	pub(crate) fn buy_execution() -> Weight {
+		(7_095_000 as Weight)
+	}
+	// Storage: PolkadotXcm Queries (r:1 w:0)
+	pub(crate) fn query_response() -> Weight {
+		(13_270_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(1 as Weight))
+	}
+	pub(crate) fn transact() -> Weight {
+		(16_375_000 as Weight)
+	}
+	pub(crate) fn refund_surplus() -> Weight {
+		(7_319_000 as Weight)
+	}
+	pub(crate) fn set_error_handler() -> Weight {
+		(3_515_000 as Weight)
+	}
+	pub(crate) fn set_appendix() -> Weight {
+		(3_501_000 as Weight)
+	}
+	pub(crate) fn clear_error() -> Weight {
+		(3_459_000 as Weight)
+	}
+	pub(crate) fn descend_origin() -> Weight {
+		(4_319_000 as Weight)
+	}
+	pub(crate) fn clear_origin() -> Weight {
+		(3_511_000 as Weight)
+	}
+	// Storage: PolkadotXcm SupportedVersion (r:1 w:0)
+	// Storage: PolkadotXcm VersionDiscoveryQueue (r:1 w:1)
+	// Storage: PolkadotXcm SafeXcmVersion (r:1 w:0)
+	// Storage: ParachainSystem HostConfiguration (r:1 w:0)
+	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
+	pub(crate) fn report_error() -> Weight {
+		(13_284_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(5 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	// Storage: PolkadotXcm AssetTraps (r:1 w:1)
+	pub(crate) fn claim_asset() -> Weight {
+		(7_985_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(1 as Weight))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	pub(crate) fn trap() -> Weight {
+		(3_515_000 as Weight)
+	}
+	// Storage: PolkadotXcm VersionNotifyTargets (r:1 w:1)
+	// Storage: PolkadotXcm SupportedVersion (r:1 w:0)
+	// Storage: PolkadotXcm VersionDiscoveryQueue (r:1 w:1)
+	// Storage: PolkadotXcm SafeXcmVersion (r:1 w:0)
+	// Storage: ParachainSystem HostConfiguration (r:1 w:0)
+	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
+	pub(crate) fn subscribe_version() -> Weight {
+		(16_657_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(6 as Weight))
+			.saturating_add(T::DbWeight::get().writes(3 as Weight))
+	}
+	// Storage: PolkadotXcm VersionNotifyTargets (r:0 w:1)
+	pub(crate) fn unsubscribe_version() -> Weight {
+		(5_622_000 as Weight)
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: ParachainInfo ParachainId (r:1 w:0)
+	// Storage: PolkadotXcm SupportedVersion (r:1 w:0)
+	// Storage: PolkadotXcm VersionDiscoveryQueue (r:1 w:1)
+	// Storage: PolkadotXcm SafeXcmVersion (r:1 w:0)
+	// Storage: ParachainSystem HostConfiguration (r:1 w:0)
+	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
+	pub(crate) fn initiate_reserve_withdraw() -> Weight {
+		(878_786_000 as Weight)
```

Comment:
same thing here. This is high for westmint and statemine but not statemint....?

Reply:
needs a regen